src/app/pages/TokenDashboardPage/TokenTotalTransactionsCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { SnapshotCard } from '../../components/Snapshots/SnapshotCard'
+import { COLORS } from '../../../styles/theme/colors'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useLoaderData } from 'react-router-dom'
+import { useAccount } from '../AccountDetailsPage/hook'
+
+export const TokenTotalTransactionsCard: FC = () => {
+  const { t } = useTranslation()
+  const scope = useRequiredScopeParam()
+
+  const address = useLoaderData() as string
+
+  const { isFetched, account } = useAccount(scope, address)
+  const value = account?.stats.num_txns
```

Comment:
`.toLocaleString()` would be nice, the same way you are using it above.

Reply:
No need, since it will be formatted using i18next later in the code.